### PR TITLE
Create Knative Social Media Guidelines : https://github.com/knative/community/issues/1444

### LINKED ISSUE #1444: Create social media guidelines

### PR DESCRIPTION
**Pull request by nainaz**
Fixes #1444 

# Changes
New markdown for social media guidelines

-
-
-

/kind <kind>

Fixes #

**Release Note**

```release-note

```

**Docs**

```docs

```

### CODE REVIEW

**Review by csantanapr — CHANGES_REQUESTED** [review]
make the file .md

File: Knative Social Media Guidelines.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+### Strategy
+
+- Keep messages positive and uplifting, consistent with the values and principles of CNCF.
+- Communicate big picture ideas vs “announcements” or news. News will be positioned less like an announcement and more around what it means.
+- Share content across social media channels that benefit the ecosystem as a whole.
+- Maintain a balance of posting on CNCF activities/news and ecosystem-focused content.
+- Share vendor-neutral, community-sourced posts that are informational, engaging, and ecosystem-focused.
+- Engage with the community through retweets and sharing of community content.
+
+### General Guidelines
```
Comment (aliok) [inline Knative Social Media Guidelines.md:34]: Would you want to mention anything around the frequency? I am not an expert in social media, but we wouldn't want to spam followers with very frequent posts. Similarly, we wouldn't want to post too infrequently.

In some groups, I've seen that they decide on a social media post every community meeting. Perhaps we can introduce something like that...

- In every Knative SC meeting (happens every 2 weeks), the SC can decide on what to post next week and the week after.
- With folks assigned to the tasks of creating a blurb for the post.

**Comment by aliok** [comment]
Another thing, could you please rename the file to something like `SOCIAL_MEDIA_GUIDELINES.md`?
Other files have that kind of pattern.

One last thing, when the PR merges, can you delete the `nainaz-patch-1` branch? I know it is a bit confusing, but generally:
- You work in your own fork
- Create a branch there
- Send a PR agains main branch of this repo

**Review by sandipanpanda — COMMENTED** [review]
I hope you will find this useful: https://github.com/kubernetes/community/blob/master/communication/contributor-comms/blogging-resources/social-guidelines.md

**Comment by aliok** [comment]
@nainaz now that we want to use social media more often, would you like me to take this PR over?

**Comment by nainaz** [comment]
Please go ahead

**Comment by sandipanpanda** [comment]
I am eager to use my experience as a Contributor Communications team member of Kubernetes to help Knative's social media efforts, should you need any help.

File: Knative Social Media Guidelines.md (hunk 1 above)
Comment (evankanderson) [reply Knative Social Media Guidelines.md:34]: It feels like an ideal cadence might be weekly or twice-weekly, but I'm not sure we're up for producing content that fast yet.

**Comment by nainaz** [comment]
Agree with weekly cadence and I created an issue to add Linkedin added to
Follow us on website

Thank you,
-N

**Comment by aliok** [comment]
@knative/steering-committee 

I've updated the document. Can you review please?

File: SOCIAL-MEDIA.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+## Regular posts
+
+### New contributor celebration
+
+Every month, we celebrate new contributors to the Knative project. We will share a post on Twitter and LinkedIn to celebrate the new contributors.
+See the [Knative contributor data](https://github.com/aliok/knative-contrib-report?tab=readme-ov-file#list-of-all-contributors-to-the-knative-project-2) for more information on how to find new contributors.
+
+### Biweekly social media post
+
+After each Knative Steering Committee meeting, we will share a post on Twitter and LinkedIn to share the highlights of the past 2 weeks.
```
Comment (evankanderson) [inline SOCIAL-MEDIA.md:73]: This seems like it might veer into the "announcements" or "news" aspect.  Other that that, this LGTM.

**Review by evankanderson — APPROVED** [review]

**Pull request merged**